common/ts/__tests__/publish-task-test.ts

```diff
@@ -72,14 +82,29 @@ it('check multiple report status and set global quality gate for build propertie
     null
   );
 
-  jest.spyOn(Analysis, 'getAnalysis').mockImplementationOnce(() => returnedAnalysisOk);
-  jest.spyOn(Analysis, 'getAnalysis').mockImplementationOnce(() => returnedAnalysisOk);
+  jest.spyOn(Analysis, 'getAnalysis').mockImplementationOnce(
```

Comment:
There is also a `jest.spyOn(Analysis, 'getAnalysis').mockResolvedValueOnce(returnedAnalysisOk);`